packages/xdl/src/Project.ts

```diff
@@ -354,8 +376,10 @@ export async function exportForAppHosting(
     dumpAssetmap?: boolean;
     dumpSourcemap?: boolean;
     publishOptions?: PublishOptions;
-  } = {}
+  } = {},
+  eas: boolean
 ): Promise<void> {
+  const absoluteOutputDir = path.resolve(process.cwd(), outputDir);
```

Comment:
This used to be called right before `ExportAppAsync`. I brought it inside and replaced the relevant `outputDir` with `absoluteOutputDir`.